### PR TITLE
Issue #SC-2200 feat: removed auth for question/questionset read api

### PR DESCRIPTION
**Pull request by krgauraw**

### CODE REVIEW

File: ansible/roles/kong-api/defaults/main.yml [hunk 1]
```diff
@@ -5543,9 +5543,6 @@ kong_apis:
       - name: jwt
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5543]: if no authorization is required remove the jwt.

File: ansible/roles/kong-api/defaults/main.yml [hunk 2]
```diff
@@ -5669,9 +5666,6 @@ kong_apis:
...
```
Comment (SMYALTAMASH) [inline ansible/roles/kong-api/defaults/main.yml:5666]: if no authorization is required remove the jwt.

File: ansible/roles/kong-api/defaults/main.yml (hunk 1 above)
Comment (krgauraw) [reply ansible/roles/kong-api/defaults/main.yml]: done

File: ansible/roles/kong-api/defaults/main.yml (hunk 2 above)
Comment (krgauraw) [reply ansible/roles/kong-api/defaults/main.yml]: done

**Review by SMYALTAMASH — APPROVED** [review]

**Pull request merged**